Dockerfile

```diff
@@ -6,26 +6,26 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-FROM alpine:3.13.5
-
-ARG KUBERNETES_VERSION=1.21.1
-ARG REST_SERVER_VERSION=0.10.0
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/go-toolset
+FROM registry.access.redhat.com/ubi8/go-toolset:1.15.7-11 as builder
+ENV GOPATH=/tmp/go/
 
+ENV REST_SERVER_TAG=v0.10.0
 RUN export ARCH="$(uname -m)" && \
     if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; \
     elif [[ ${ARCH} == "aarch64" ]]; then export ARCH="arm64"; \
     fi && \
-    apk add --no-cache curl && \
-    cd /usr/local/bin && \
-    curl -sLO https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/${ARCH}/kubectl && \
-    chmod +x kubectl && \
-    cd /tmp && \
-    set -x && \
-    curl -sLO https://github.com/restic/rest-server/releases/download/v${REST_SERVER_VERSION}/rest-server_${REST_SERVER_VERSION}_linux_${ARCH}.tar.gz && \
-    tar -xzf rest-server_${REST_SERVER_VERSION}_linux_${ARCH}.tar.gz && \
-    cp rest-server_${REST_SERVER_VERSION}_linux_${ARCH}/rest-server /usr/local/bin/rest-server && \
-    rm -rf rest-server_${REST_SERVER_VERSION}_linux_${ARCH} rest-server_${REST_SERVER_VERSION}_linux_${ARCH}.tar.gz && \
-    apk del curl
+    mkdir -p $GOPATH && cd $GOPATH && \
+    git clone --depth 1 --branch $REST_SERVER_TAG https://github.com/restic/rest-server.git && \
+    cd rest-server && \
+    go mod vendor && \
+    GOOS=linux GOARCH=${ARCH} CGO_ENABLED=0 go build -mod=vendor -o rest-server  ./cmd/rest-server
+
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/ubi-micro
+FROM registry.access.redhat.com/ubi8/ubi-micro:8.4-72
+
+COPY --from=builder /tmp/go/rest-server/rest-server /usr/local/bin/rest-server
+COPY --from=builder /tmp/go/rest-server/LICENSE /usr/local/bin/rest-server-LICENSE.txt
```

Comment:
do we need to copy the license in the image ?

Reply:
Yes, this is the requirement, see https://github.com/eclipse/che/issues/20028